src/main/java/com/alex/eshop/restcontroller/ItemController.java

```diff
@@ -21,12 +22,16 @@ public List<ItemDTO> lastFiveItems() {
     }
 
     @GetMapping("items/{id}")
-    public ItemDTO getItem(@PathVariable Long id) {
+    public ItemDTO getItem(@PathVariable Long id){
         return itemService.getItemWithCategoryInfo(id);
     }
 
     @GetMapping("/items")
-    public List<ItemDTO> getItemWithCategory(@RequestParam Long categoryId) {
-        return itemService.getItemsInCategory(categoryId);
+    public List<ItemDTO> getItemWithCategory(@RequestParam Long categoryId,
+                                             @RequestParam(value = "page", defaultValue = "0") int page,
```

Comment:
You don't need separate **page**, **size**, **sort** parameters. Use only **Pageable pageable**. It includes all these params